src/status_im2/common/resources.cljs

```diff
@@ -47,7 +47,8 @@
    :no-contacts-to-invite-light   (js/require "../resources/images/ui2/no-contacts-to-invite-light.png")
    :no-contacts-to-invite-dark    (js/require "../resources/images/ui2/no-contacts-to-invite-dark.png")
    :no-notifications-light        (js/require "../resources/images/ui2/no-notifications-light.png")
-   :no-notifications-dark         (js/require "../resources/images/ui2/no-notifications-dark.png")})
+   :no-notifications-dark         (js/require "../resources/images/ui2/no-notifications-dark.png")
+   :div                           (js/require "../resources/images/ui2/div.png")})
```

Comment:
should be an icon? 🤔 or perhaps it's better we just implement this in code? it seems quite simple. Maybe you can check with design team if you haven't already  👍 i.e if this is just a static part of the design it seems better to just a styled view component

Reply:
@J-Son89 yeah that's right it is very simple it can be a `rn/view`. What about the token image? We can pass `(resources/tokens token)` as a second argument, but it would be more flexible if we can read the resources file from quo2. The resources file is just static data -- wouldn't it be better if we move it to quo2?

Reply:
@OmarBasem I believe there is a resources file in quo2. quo2.foundations.resources. So yeah we can add these resources there. But still I think the resource lookup and name should not be joined because these resources have uses beyond this component